RNA-seq/03-gastric_cancer_tximport.Rmd

```diff
@@ -74,34 +74,31 @@ names(sf_files) <- sample_names
 
 ### Read in tx to gene mapping
 
-There is no header in this file.
-
-```{r}
-gene2tx_df <- readr::read_tsv(gene2tx_file, col_names = FALSE)
-gene2tx_df
-```
-
-Name the columns. 
-
 ```{r}
-colnames(gene2tx_df) <- c("gene", "transcript")
+tx2gene_df <- readr::read_tsv(tx2gene_file)
 ```
 
-`tximport` requires the `data.frame` used for mapping 
+Let's take a look at the top of this file with `head`.
 
 ```{r}
-tx2gene_df <- gene2tx_df[, c("transcript", "gene")]
-# remove gene, transcript data.frame
-rm(gene2tx_df)
+head(tx2gene_df)
 ```
 
 ### tximport main function
 
 We'll summarize to the gene level.
+Notice that `tx2gene_df` uses identifiers that don't include [Ensembl version information](https://useast.ensembl.org/Help/Faq?id=488) (e.g., don't end in `.1`).
```

Comment:
```suggestion
Notice that `tx2gene_df` uses identifiers that don't include [Ensembl version information](https://useast.ensembl.org/Help/Faq?id=488) (e.g., don't end in a decimal point).
```

Reply:
These don't end in decimal points. They end in a decimal point followed by a version number. I can probably tweak this a bit to be more clear.